### PR TITLE
Create a benefits of speaking post

### PR DESCRIPTION
**Pull request by tim-schilling**

### CODE REVIEW

**Comment by tim-schilling** [comment]
I switched the photo:

Screen Shot 2024-03-31 at 8 05 17 AM

File: _posts/2024-04-02-benefits-of-speaking-djangocon.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+author: DjangoCon US Communication Team
```
Comment (drewbrew) [inline _posts/2024-04-02-benefits-of-speaking-djangocon.md]:
```suggestion
author: DjangoCon US Communications Team
```

File: _posts/2024-04-02-benefits-of-speaking-djangocon.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- **Career advancement:** Speaking at DjangoCon US can help you advance your career by demonstrating your knowledge and expertise, which can lead to new job opportunities or promotions.
+- **Feedback and learning:** Presenting at DjangoCon US gives you the opportunity to receive feedback from your peers, which helps you improve your skills and knowledge.
+- **Giving back to the community:** By sharing your knowledge and experience with others at DjangoCon US, you are contributing to the growth and development of the Django community, which is a rewarding experience in itself.
+- **Travel and conference experience:** Speaking at DjangoCon US includes travel to interesting locations, attending the conference to learn from other presenters, and give you a valuable conference experience that broadens your horizons.
+
+## Need Topic Ideas?
+
+We've had organizers create lists of talk ideas they'd like to see:
+- [Kati's list](https://katherinemichel.github.io/portfolio/djangocon-us-2024-topics-inspiration-list.html)
+- [Drew's list](https://winstel.dev/2024/03/01/talk-ideas-for-dcus-24/)
```
Comment (drewbrew) [inline _posts/2024-04-02-benefits-of-speaking-djangocon.md]: /ping @jefftriplett as a nudge

**Review by drewbrew — APPROVED** [review]

File: _posts/2024-04-02-benefits-of-speaking-djangocon.md (hunk 2 above)
Comment (jefftriplett) [reply _posts/2024-04-02-benefits-of-speaking-djangocon.md]: @drewbrew thanks. I'll finish mine this week, but no reason to wait on it.

**Pull request merged**